lib/travis/api/v3/models/build.rb

```diff
@@ -8,7 +8,8 @@ class Models::Build < Model
     has_many :jobs,
       foreign_key: :source_id,
       order:       :id,
-      dependent:   :destroy
+      dependent:   :destroy,
+      class_name:  'Travis::API::V3::Models::Job'.freeze
```

Comment:
Probably not required, but it seemed the other explicitly defined relations had this.